owncloudApp/src/main/java/com/owncloud/android/presentation/previews/PreviewTextViewModel.kt

```diff
@@ -38,11 +40,17 @@ class PreviewTextViewModel(
     private val filterFileMenuOptionsUseCase: FilterFileMenuOptionsUseCase,
     private val contextProvider: ContextProvider,
     private val coroutinesDispatcherProvider: CoroutinesDispatcherProvider,
+    getFileByIdAsStreamUseCase: GetFileByIdAsStreamUseCase,
```

Comment:
We usually group all the use cases together in the parameters, and then the left ones 👍 (applicable to all ViewModels in which you introduced new parameters)

Reply:
Changed 👍